.github/workflows/crowdin-download.docs.yml

```
@@ -294,9 +254,6 @@ jobs:
       - name: Create PR
         uses: freecodecamp/crowdin-action@main
         env:
-          CROWDIN_PERSONAL_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_SERVICE_TOKEN }}
-          CROWDIN_API_URL: 'https://freecodecamp.crowdin.com/api/v2/'
-          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID_DOCS }}
           PLUGIN: 'pull-request'
           GH_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_PAT }}
```

```suggestion
          GH_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_PAT }}
```

Can this be renamed to make use of `GITHUB_TOKEN`?

---

No. The `GITHUB_TOKEN` option can't trigger CI.

> When you use the repository's GITHUB_TOKEN to perform tasks, events triggered by the GITHUB_TOKEN will not create a new workflow run.

Which means the PR wouldn't have checks running on it.